cli/src/pcluster/models/cluster.py

```diff
@@ -527,6 +543,20 @@ def __init__(
 # ---------------------- Others ---------------------- #
 
 
+class Tag(BaseTag):
+    """Represent the Tag configuration."""
+
+    def __init__(
+        self,
+        key: str = None,
+        value: str = None,
+    ):
+        super().__init__(key, value)
+
+    def _register_validators(self):
+        self._add_validator(TagKeyValidator, key=self.key)
```

Comment:
Because it's used by ImageBuilder too and there are other conditions to be evaluated.
E.g. in ImageBuilder they have PclusterVersion or other names for the default tags. We can put something in common by passing the forbidden_keys but I'd let @yuleiwan do it when addressing it.

Reply:
thanks for the consideration, but I add tag validate in syntactic validate, should we move it to semantic validate?

Reply:
Let's figure out if we want to use PclusterVersion or Version